media-sound/teamspeak-server-bin/teamspeak-server-bin-3.1.1.ebuild

```diff
@@ -0,0 +1,134 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eutils multilib systemd user
+
+DESCRIPTION="TeamSpeak is software for quality voice communication via the Internet"
+HOMEPAGE="https://www.teamspeak.com/"
+SRC_URI="amd64? ( http://ftp.4players.de/pub/hosted/ts3/releases/${PV}/teamspeak3-server_linux_amd64-${PV}.tar.bz2 )
+	x86? ( http://ftp.4players.de/pub/hosted/ts3/releases/${PV}/teamspeak3-server_linux_x86-${PV}.tar.bz2 )"
+
+LICENSE="teamspeak3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc mysql tsdns"
+
+RDEPEND="mysql? ( dev-db/mariadb-connector-c )"
+
+RESTRICT="fetch mirror strip"
+
+S="${WORKDIR}/teamspeak3-server_linux_${ARCH}"
+
+pkg_nofetch() {
+	elog "Please download ${A} from ${HOMEPAGE}"
+	elog "and place it in your DISTDIR directory."
+}
+
+pkg_setup() {
+	enewuser teamspeak
+}
+
+src_install() {
+	local TS3_DIR="teamspeak3-server"
+	local TS3_UG="teamspeak"
+
+	# Accept license
+	touch "${T%/}"/.ts3server_license_accepted || die
+	insinto "/opt/${TS3_DIR}"
+	doins "${T}"/.ts3server_license_accepted
+
+	# Install binary and wrapper
+	exeinto "/opt/${TS3_DIR}"
+	doexe "ts3server"
+	insinto "/opt/${TS3_DIR}"
+	doins "${FILESDIR}/ts3server-bin"
```

Comment:
Is there a reason you're not making the file executable (i.e. using `doexe`)?

Reply:
No specific reason, I will change it, it's even better this way.